std_msgs/String item
---
std_msgs/Header header
geometry_msgs/Pose pose
bool found